app/src/main/java/com/example/aurora/ForestThemeActivity.java

```diff
@@ -0,0 +1,44 @@
+package com.example.aurora;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+
+import android.graphics.drawable.AnimatedVectorDrawable;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ImageView;
+
+public class ForestThemeActivity extends AppCompatActivity {
+
+    private AnimatedVectorDrawable blob;
+    private AnimatedVectorDrawable pause;
+    private AnimatedVectorDrawable play;
+    private ImageView blobView;
+    private ImageView pp_button;
+    private boolean tick = true;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_forest_theme);
+
+
+        FragmentManager fm = getSupportFragmentManager();
```

Comment:
This chunk of code is duplicated three times in this project. Try to avoid code duplication.